src/Nav.js

```diff
@@ -22,21 +22,15 @@ function Nav({
   const queryParams = useQuery();
   const searchQuery = queryParams.get("q");
   useEffect(() => {
+    const focusSearchBar = (event) => {
+      if (document.activeElement === document.body) {
+        event.preventDefault(); // Search shortcuts
+        document.getElementById("search-bar").focus();
+      } // should only work when the document has focus.
+    };
     const unsubscribe = tinykeys(window, {
-      "$mod+KeyK": (event) => {
-        // Search shortcuts should only work when the
-        if (document.activeElement === document.body) {
-          event.preventDefault(); // document has focus.
-          document.getElementById("search-bar").focus();
-        }
-      },
-      Slash: (event) => {
-        // Search shortcuts should only work when the
-        if (document.activeElement === document.body) {
-          event.preventDefault(); // document has focus.
-          document.getElementById("search-bar").focus();
-        }
-      },
+      "$mod+KeyK": (event) => focusSearchBar(event),
+      Slash: (event) => focusSearchBar(event),
```

Comment:
`(event) => focusSearchBar(event),` is a function that calls a function and passes the same value, which is redundant.

```suggestion
      Slash: focusSearchBar,
```